src/parachain/loans.ts

```diff
@@ -375,16 +319,139 @@ export class DefaultLoansAPI implements LoansAPI {
             })
         );
 
-        return <LendPosition[]>allMarketsPositions.filter(position => position !== null);
+        return <LendPosition[]>allMarketsPositions.filter((position) => position !== null);
     }
 
     getBorrowPositionsOfAccount(accountId: AccountId): Promise<Array<BorrowPosition>> {
         return Promise.resolve(MOCKDATA_BORROW_POSITIONS);
     }
 
+    async _getLoanAssetLendApy(underlyingCurrencyId: InterbtcPrimitivesCurrencyId): Promise<Big> {
+        const rawLendApy = await this.api.query.loans.supplyRate(underlyingCurrencyId);
+
+        // Return percentage
+        return decodeFixedPointType(rawLendApy).mul(100);
+    }
+    async _getLoanAssetBorrowApy(underlyingCurrencyId: InterbtcPrimitivesCurrencyId): Promise<Big> {
```

Comment:
Also here and in the other places with `LoanAsset` in their name
```suggestion
    async _getBorrowApy(underlyingCurrencyId: InterbtcPrimitivesCurrencyId): Promise<Big> {
```